mkosi/__init__.py

```diff
@@ -6590,7 +6628,15 @@ def setup_ssh(
         return None
 
     if args.distribution in (Distribution.debian, Distribution.ubuntu):
-        unit = "ssh"
+        unit = "ssh.socket"
+
+        if args.ssh_port != 22:
+            add_dropin_config(root, unit, "port",
+                      f"""\
+                      [Socket]
+                      ListenStream=
+                      ListenStream={args.ssh_port}
+                      """)
```

Comment:
```suggestion
            add_dropin_config(root, unit, "port",
                              f"""\
                              [Socket]
                              ListenStream=
                              ListenStream={args.ssh_port}
                              """) 
```
I think this identation will make @keszybz happier :) (me, too for that matter)

Reply:
updated

Reply:
Your speed always amazes me. Such roundtrip times make working a real pleasure. :)

Reply:
Fastest `git push` in the West!